contents/customers/brainboard.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: How Brainboard raised conversion rates from 20% to 70% with PostHog
```

Comment:
Having two numbers in the title makes it slightly hard to parse. Increased conversion rates 3x or 350%?

Reply:
Oh, that's such a good point - thanks! <3